content/courses/Guidelines_for_Safe_and_Secure_Ada_SPARK/chapters/guidelines/object_oriented_programming/oop04_limit_statically-dispatched_calls_to_primitive_operations.rst

```diff
@@ -0,0 +1,116 @@
+-------------------------------------------------------------------
+Limit Statically-Dispatched Calls To Primitive Operations (OOP04)
+-------------------------------------------------------------------
+
```

Comment:
capitalisation of prepositions in guidance names are not consistent e.g. here "To" is capitalised but in "Prove Absence of Run-time Exceptions (EXU04)" the prepositions "of" is not capitalised.

Reply:
prepositions and articles should be lower case (unless first or last word in title)